src/index.js

```diff
@@ -80,3 +80,6 @@ const initOnce = (fn, message) => (...args) => {
 };
 
 export const init = initOnce(_init, 'FullStory init has already been called once, additional invocations are ignored');
+
+// normalize undefined into boolean
+export const isInitialized = () => !!window._fs_initialized;
```

Comment:
you could probably do away with the global entirely by just checking `!!fs()` here and on line 73?

Reply:
Originally made this change, but I think semantically using `_fs_initialized` might be closer to the truth? based on this github issue https://github.com/fullstorydev/fullstory-browser-sdk/issues/115, people are using this it directly for this purpose, so we would need to support the global indefinitely in any case 😅